wit/wasmcloud/lattice-control/lattice-control.wit

```diff
@@ -0,0 +1,560 @@
+      // which will turn this member into `annotations: Vec<HashMap<String, String>>`
+      annotations-map: list<tuple<string,string>>,
+    }
+
+    /// A request to stop the given host
+    record stop-host-command {
+      /// The ID of the lattice for this request
+      lattice-id: string,
+
+      /// The ID of the host to stop
+      host-id: string,
+
+      /// An optional timeout (in seconds)
+      timeout: option<u64>,
+    }
+
+    /// A command requesting a host to perform a live update on an indicated actor,
+    /// supplying a new image reference.
+    ///
+    /// Note that live updates are only possibly through image references.
+    record update-actor-command {
+      /// The ID of the lattice for this request
+      lattice-id: string,
+
+      /// The host ID on which to perform the live actor update
+      host-id: string,
+
+      /// The public key of the actor to update
+      actor-id: string,
+
+      /// The new image reference of the upgraded version of this actor
+      new-actor-ref: string,
+
+      /// Optional set of annotations used to describe the nature of this update request.
+      ///
+      /// Only actor instances that have matching annotations will be upgraded, allowing for instance
+      /// isolation by autonomous agent deployment spec (ex. wadm), for example.
+      //
+      // This member expects/requires the use of bindgen's `replace_witified_maps` option,
+      // which will turn this member into `annotations: Vec<HashMap<String, String>>`
+      annotations-map: option<list<tuple<string,string>>>,
+    }
+
+    /// Standard response for control interface operations
+    record ctl-operation-ack {
+      /// Whether the operation was accepted
+      accepted: bool,
+
+      /// If an error occurred, a string representing the error
+      error: string,
+    }
+
+    /// Representation of a wasmcloud Host
+    record host {
+      /// The ID of the host
+      id: string,
+
+      /// Uptime in seconds of the host
+      uptime-seconds: u64,
+
+      /// Human-friendly uptime description
+      uptime-human: option<string>,
+
+      /// map of key-value pairs which serve as labels of the host
+      //
+      // This member expects/requires the use of bindgen's `replace_witified_maps` option,
+      // which will turn this member into `annotations: Vec<HashMap<String, String>>`
+      labels-map: option<list<tuple<string,string>>>,
+
+      /// Human-friendly uptime description
+      version: option<string>,
+
+      /// Comma-delimited list of valid cluster issuer public keys known to this host
+      cluster-issuers: option<string>,
+
+      /// JetStream domain (if applicable) in use by this host
+      js-domain: option<string>,
+
+      /// NATS server host used for the control interface
+      ctl-host: option<string>,
+
+      /// NATS server host used for provider RPC
+      ///
+      /// This field has been deprecated and is now equivalent to
```

Comment:
Super duper nit: Some comments are wrapped at 80 chars or less, others at 100, and some not at all. Consider standardizing to the 100 chars that seems to be common in our code

Reply:
- [ ] consistent comment wrapping @ 100 chars